src/core/src/collection.rs

```diff
@@ -129,6 +137,15 @@ impl Collection {
         })
     }
 
+    #[cfg(all(feature = "branchwater", not(target_arch = "wasm32")))]
+    pub fn from_rocksdb<P: AsRef<Path>>(dirname: P) -> Result<Self> {
+        let path = dirname.as_ref().as_str().to_string();
```

Comment:
Let's move the import into the function, just to make it easier to track future `cfg` changes in one place instead of two:

```suggestion
    pub fn from_rocksdb<P: AsRef<Path>>(dirname: P) -> Result<Self> {
        use crate::index::revindex::{ RevIndex, RevIndexOps };
    
        let path = dirname.as_ref().as_str().to_string();
```